### ISSUE TITLE
Do research on current the state of the art in Distributed SGD using GPU framework

### ISSUE THREAD

**Issue by qq95113** [qq95113/RecSys_Preparation#28]
https://towardsdatascience.com/accelerating-deep-learning-using-distributed-sgd-an-overview-e66c4aee1a0c
https://github.com/tmulc18/Distributed-TensorFlow-Guide

https://ray.readthedocs.io/en/latest/index.html
Ray is a flexible, high-performance distributed execution framework.
Ray includes an implementation of synchronous distributed stochastic gradient descent (SGD), which is competitive in performance with implementations in Horovod and Distributed TensorFlow.
https://www.oreilly.com/ideas/distributed-tensorflow
https://arxiv.org/pdf/1711.05979.pdf
https://openreview.net/pdf?id=D1VDZ5kMAu5jEJ1zfEWL
